### PR TITLE
Adds benchmark tests for Schematics tool

### PR DESCRIPTION
**Pull request by AlvaroLQueiroz**
## Change Summary

This PR adds a comparison tests for the [Schematic](https://schematics.readthedocs.io/en/latest/index.html) tool.

### CODE REVIEW

**Comment by samuelcolvin** [comment]
Normally I refuse new benchmarks, but I do see that this library is relatively popular.

Could you give a summary of the benchmark results with this added? I'm afraid I don't have a linux machine at hand right now.

File: benchmarks/test_schematics.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            skills = ListType(ModelType(Skill), default=[])
+
+
+        self.model = Model
+
+    def validate(self, data):
+        try:
+            obj = self.model(data)
+            return True, obj.validate()
+        except Exception as e:
```
Comment (samuelcolvin) [inline benchmarks/test_schematics.py:49]: surely not all exceptions should be caught here?

Comment (AlvaroLQueiroz) [reply benchmarks/test_schematics.py:49]: You're right. I've already corrected.

**Comment by AlvaroLQueiroz** [comment]
@samuelcolvin i don't have a linux machine too. But this is the results executed on a Mac Pro with MacOS Catalina, i5 2GHZ, 16GB.

Package | Version | Relative Performance | Mean validation time
--- | --- | --- | ---
pydantic | `1.7.3` |  | 91.8μs
attrs + cattrs | `20.3.0` | 1.3x slower | 116.6μs
valideer | `0.4.2` | 1.3x slower | 122.0μs
marshmallow | `3.10.0` | 2.5x slower | 225.0μs
voluptuous | `0.12.1` | 2.5x slower | 226.0μs
trafaret | `2.1.0` | 3.3x slower | 305.0μs
schematics | `2.1.0` | 10.3x slower | 948.6μs
django-rest-framework | `3.12.2` | 12.6x slower | 1154.3μs
cerberus | `1.3.2` | 22.6x slower | 2073.4μs

**Pull request merged**